docs/source/starter/style_guide.rst

```diff
@@ -173,55 +151,57 @@ In practice, this code looks like:
 
 Forward vs training_step
 ========================
-
-We recommend using forward for inference/predictions and keeping ``training_step`` independent.
+We recommend using forward for inference/predictions and keeping training_step independent
 
 .. code-block:: python
 
     def forward(self, x):
         embeddings = self.encoder(x)
-        return embeddings
 
 
-    def training_step(self, batch, batch_idx):
-        x, y = batch
+    def training_step(self):
+        x, y = ...
         z = self.encoder(x)
         pred = self.decoder(z)
         ...
 
+However, when using DataParallel, you will need to call forward manually
+
+.. code-block:: python
+
+    def training_step(self):
+        x, y = ...
+        z = self(x)  # < ---------- instead of self.encoder(x)
+        pred = self.decoder(z)
+        ...
 
 --------------
 
 ****
 Data
 ****
-
 These are best practices for handling data.
 
 Dataloaders
 ===========
-
-Lightning uses :class:`~torch.utils.data.DataLoader` to handle all the data flow through the system. Whenever you structure dataloaders,
+Lightning uses dataloaders to handle all the data flow through the system. Whenever you structure dataloaders,
```

Comment:
Let's keep :class:`~torch.utils.data.DataLoader`. The more connection to PyTorch Objects, the more transparent it is for the users.